geomet_data_registry/layer/radar_1km.py

```diff
@@ -75,30 +74,31 @@ def identify(self, filepath):
         LOGGER.debug('Defining the different file properties')
         self.wx_variable = file_pattern_info['wx_variable']
 
-        if self.wx_variable not in file_dict[self.model]['variable']:
+        if self.wx_variable not in self.file_dict[self.model]['variable']:
             msg = 'Variable "{}" not in ' \
                   'configuration file'.format(self.wx_variable)
             LOGGER.warning(msg)
             return False
 
         time_format = '%Y%m%d%H%M'
-        date_ = datetime.strptime(file_pattern_info['time_'], time_format)
+        self.date_ = datetime.strptime(file_pattern_info['time_'], time_format)
 
-        layer_name = file_dict[self.model]['variable'][self.wx_variable]['geomet_layer']  # noqa
+        self.layer_name = self.file_dict[self.model]['variable'][self.wx_variable]['geomet_layer']  # noqa
 
-        member = file_dict[self.model]['variable'][self.wx_variable]['member']  # noqa
-        elevation = file_dict[self.model]['variable'][self.wx_variable]['elevation']  # noqa
+        member = self.file_dict[self.model]['variable'][self.wx_variable]['member']  # noqa
+        elevation = self.file_dict[self.model]['variable'][self.wx_variable]['elevation']  # noqa
         str_fh = re.sub('[^0-9]',
                         '',
-                        date_.strftime('%Y-%m-%dT%H:%M:%SZ'))
-        identifier = '{}-{}'.format(layer_name, str_fh)
+                        self.date_.strftime('%Y-%m-%dT%H:%M:%SZ'))
```

Comment:
let's set a global val in `geomet_data_registry.util`, so we can have a single, consistent value we can import like:

```python
# in geomet_data_registry/util.py
DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
```
Then from anywhere else:

```python
from geomet_data_registry.util import DATE_FORMAT
```

Reply:
Changed in the code only for radar. 

I am working on the NWP models at the moment and to avoid conflicts, I will update the DATE_FORMAT in the next merge request.

This is fixed, so tagging as resolved